src/operator/numpy/np_elemwise_unary_op_basic.cc

```diff
@@ -82,6 +83,90 @@ NNVM_REGISTER_OP(_np_copy)
+      return std::vector<std::string>{__input_name$};                                     \
+    })                                                                                    \
+  .set_attr<FCompute>("FCompute<cpu>", UnaryOp::ComputeMixedType<cpu, __kernel$>)         \
+  .add_argument(__input_name$, "NDArray-or-Symbol", "The input array.")
+
+#define MXNET_OPERATOR_REGISTER_UNARY_MIXEDTYPE_IN(name)            \
+  NNVM_REGISTER_OP(name)                                            \
+  .set_num_inputs(2)                                                \
+  .set_num_outputs(1)                                               \
+  .set_attr<nnvm::FListInputNames>("FListInputNames",               \
+    [](const NodeAttrs& attrs) {                                    \
+      return std::vector<std::string>{"lhs", "rhs"};                \
+    })                                                              \
+  .set_attr<mxnet::FInferShape>("FInferShape", ElemwiseShape<2, 1>)  \
+  .set_attr<nnvm::FInplaceOption>("FInplaceOption",                 \
+    [](const NodeAttrs& attrs){                                     \
+      return std::vector<std::pair<int, int> >{{0, 0}, {1, 0}};     \
+    })                                                              \
+  .add_argument("lhs", "NDArray-or-Symbol", "first input")          \
+  .add_argument("rhs", "NDArray-or-Symbol", "second input")
+
+/*! \brief Binary launch */
+#define MXNET_OPERATOR_REGISTER_UNARY_MIXEDTYPE_INOUT(name)         \
+  NNVM_REGISTER_OP(name)                                            \
+  .set_num_inputs(3)                                                \
+  .set_num_outputs(1)                                               \
+  .set_attr<nnvm::FListInputNames>("FListInputNames",               \
+    [](const NodeAttrs& attrs) {                                    \
+      return std::vector<std::string>{"lhs", "rhs"};                \
+    })                                                              \
+  .set_attr<mxnet::FInferShape>("FInferShape", ElemwiseShape<3, 1>)  \
+  .set_attr<nnvm::FInplaceOption>("FInplaceOption",                 \
+    [](const NodeAttrs& attrs){                                     \
+      return std::vector<std::pair<int, int> >{{0, 0}, {1, 0}};     \
+    })                                                              \
+  .add_argument("lhs", "NDArray-or-Symbol", "first input")          \
+  .add_argument("rhs", "NDArray-or-Symbol", "second input")
+
+#define MXNET_OPERATOR_REGISTER_UNARY_MIXEDTYPE_IN_CPU(__name$, __kernel$)                    \
+  MXNET_OPERATOR_REGISTER_UNARY_MIXEDTYPE_IN(__name$)                                         \
+  .set_attr<FCompute>("FCompute<cpu>", ElemwiseBinaryOp::MixedUnaryBackwardUseInCompute<cpu,  \
+                                                                       __kernel$>)            \
+  .set_attr<FResourceRequest>("FResourceRequest",  /* For Sparse CSR */                       \
```

Comment:
seems like `FResourceRequest` is not used, you can simply get rid of this `set_attr`